src/ray/raylet/scheduling/cluster_task_manager_test.cc

```diff
@@ -62,6 +62,8 @@ class MockWorker : public WorkerInterface {
 
   WorkerID WorkerId() const { return worker_id_; }
 
+  rpc::WorkerType GetWorkerType() const { return rpc::WorkerType::WORKER; }
```

Comment:
What's the reason we need a different worker type?

Reply:
Because we would not like to schedule tasks to the IO workers. Also it is not compatible with how we pick task workers in the worker pool. Basically once a task is assigned to a task worker, it will be poped out from the queue. Do you want to build another queue while reusing the task workers for both task executing and object spilling?

Reply:
Isn't it sufficient to keep a separate list of these workers, or am I missing something? I would prefer not to introduce this new type.